backend/src/main/java/com/hyooun/wilson/common/PersistenceAdapter.java

```diff
@@ -0,0 +1,17 @@
+package com.hyooun.wilson.common;
+
+import org.springframework.core.annotation.AliasFor;
+import org.springframework.stereotype.Component;
+
+import java.lang.annotation.*;
+
+@Target({ElementType.TYPE})
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Component
+public @interface PersistenceAdapter {
+
+    @AliasFor(annotation = Component.class)
+    String value() default "";
+
+}
```

Comment:
PersistenceAdapter에 @Component 애너테이션을 추가해주세요.

```suggestion
@Component
```